src/manage-storage.js

```diff
@@ -44,13 +44,28 @@ function manage_storage(){
 	var $message = $(E("p")).appendTo($storage_manager.$main).html(
 		"Any images you've saved to your computer with <b>File > Save</b> will not be affected."
 	);
+
+	// check if there are some images in storage
+	var imagesInStorage = Object.keys(localStorage).some(k => {return k.match(/^image#/);});
+
+	$remove_all = $storage_manager.$Button("Remove All", function(){
+		for(var k in localStorage){
+			if(k.match(/^image#/)){
```

Comment:
To fix your first point. What if we had a function polling for changes in `localStorage` using `setInterval` and updated the Manage Storage table according?

Reply:
Yes, that can work, but that's a separate feature, and my suggestion still applies. (It would still come into play in between the interval (or if there was an error with the interval code))

Reply:
Sounds good. Ill update the PR so remove all only removes the images shown.